conf/flight_plans/flight_plan.dtd

```diff
@@ -65,7 +65,9 @@ ground_alt CDATA #REQUIRED
 security_height CDATA #REQUIRED
 alt CDATA #REQUIRED
 qfu CDATA #IMPLIED
-home_mode_height CDATA #IMPLIED>
+home_mode_height CDATA #IMPLIED
+max_altitude CDATA #IMPLIED
+airspace CDATA #IMPLIED>
```

Comment:
Renamed as you suggested. If you mean `home_mode_height` that one is used in https://github.com/paparazzi/paparazzi/blob/master/sw/airborne/firmwares/fixedwing/nav.c#L429